EIPS/eip-7745.md

```diff
@@ -0,0 +1,363 @@
+
+The calculations below are based on the proposed constants:
+
+|**Name**                    |**Formula**                                 |**Value** |
+|----------------------------|--------------------------------------------|----------|
+| `epoch_index` path length  | `log2(MAX_EPOCH_HISTORY)`                  | 24 hashes|
+| `map_subindex` path length | `log2(MAPS_PER_EPOCH)`                     | 6 hashes |
+| `map_index` path length    | `log2(MAX_EPOCH_HISTORY * MAPS_PER_EPOCH)` | 30 hashes|
+| `row_index` path length    | `log2(MAP_HEIGHT)`                         | 12 hashes|
+| average row length         | `AVG_BITS_PER_ROW / 8`                     | 64 bytes |
+
+Merkle proof size is estimated for three different search range lengths: single map, one epoch and 64 epochs. Note that the leaves of the binary Merkle tree are hashes of actual rows but the proof contains the actual row data which has a varying size. For these calculations the average row length is used.
+
+|**indexing order**|**1 map**            |**1 epoch**                   |**64 epochs**                      |
+|------------------|---------------------|------------------------------|-----------------------------------|
+|column-major      |(30+12)\*32+64 = 1408|24\*32+64\*(12\*32+64) = 29440|18\*32+4096\*(12\*32+64) = 1835584 |
+|row-major         |(30+12)\*32+64 = 1408|(12+24)\*32+64\*64 = 5248     |64\*((12+24)\*32+64\*64) = 335872  |
+|mixed order       |(30+12)\*32+64 = 1408|(24+12)\*32+64\*64 = 5248     |18\*32+64\*(12\*32+64\*64) = 287296|
+
+Tree update costs are estimated as a worst-case marginal cost of each additional _log value_ in terms of updated tree nodes and amount of hashed data including actual row data. Note that it is assumed here that all updated rows belong to one or two _filter maps_ and therefore the updated tree nodes leading up to these _filter maps_ are a negligible one time cost. Also note that just like in the calculations above the average encoded row data length is used for hashed data amount estimation.
+
+|**indexing order**|**updated nodes**|**hashed bytes**|
+|------------------|-----------------|----------------|
+|column-major      |12               |12\*64+64 = 832 |
+|row-major         |30+12 = 42       |42\*64+64 = 2752|
+|mixed order       |12+6 = 18        |18\*64+64 = 1216|
+
+## Backwards Compatibility
+
+The existing log filter API (`eth_getLogs`, `eth_newFilter`, `eth_getFilterLogs`, `eth_getFilterChanges`) can be implemented with the new filter data structure. Applications relying on this API can operate without any change, with a higher performance. The EVM is not affected in any way.
+
+## Test Cases
+
+TODO
```

Comment:
can you change todo references to: 
`<!-- TODO -->`

which get flagged by CI if they are missed and EIP moved to review

Reply:
thanks, it is all green now.